### ISSUE TITLE
See presented view controller through hole

### ISSUE THREAD

**Issue by cannyboy** [leostc/LHMaterialDesign#1]
Is it possible to see the presented view controller through the hole, like this:
![see-thru](https://cloud.githubusercontent.com/assets/1590436/7429810/57ef3bec-f000-11e4-95d4-b506c95c5905.png)

(perhaps it can be 'faked' by using an image of what the presented view will look like?)

**Comment by leostc** [comment]
Yeah, it's a good idea. I'll find out whether there is a better solution. Thank you.
